src/main/java/kuit/subway/domain/Line.java

```diff
@@ -14,51 +14,41 @@
 
 @Entity
 @Getter
-@NoArgsConstructor
+@Builder
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
 public class Line extends BaseTimeEntity {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(name = "line_id")
     private Long id;
 
+    private String name;
+
     private String color;
 
     private int distance;
 
-    private String name;
-
-    @OneToMany(mappedBy = "line", cascade = CascadeType.ALL)
-    private List<Station> stations = new ArrayList<>();
+    @Embedded
+    private Sections sections;
 
-    @Builder
-    public Line(String color, int distance, String name, LocalDateTime createdDate, LocalDateTime modifiedDate) {
-        super(createdDate, modifiedDate);
-        this.color = color;
-        this.distance = distance;
-        this.name = name;
+    public static Line createLine(String name, String color, int distance) {
+        return Line.builder()
+                .name(name)
+                .color(color)
+                .distance(distance)
+                .build();
     }
-
     // 연관관계 메서드
-    public void addStations(List<Station> stations) {
-        stations.forEach(station -> station.addLine(this));
+    public void addSection(Sections sections) {
+        this.sections = sections;
     }
```

Comment:
스터디 때 언급했던 것 같은데, 보통 oneToMany 관계에 있는 List는 add나 delete 형식으로 더해주고 이렇게 아예 변경하는 것을 지양해야해요. 이유는 상진형 pr에 있습니다!
```suggestion
    public void addSection(Section section) {
        this.sections.addSection(section);
    }
```